### PR TITLE
Zombeavers as aquatic type zombie

### PR DESCRIPTION
**Pull request by nexusmrsep**
```SUMMARY: None```

#### Purpose of change
Quick fix for missing beavers in #32552

### CODE REVIEW

**Comment by cozardi** [comment]
Are beavers big enough to trigger blob reanimation?

**Comment by nexusmrsep** [comment]
Sure thing, beaver is a size of a dog and a chunky one. Mind you, they were added some time ago, I'm just changing their faction here so they may attack fish.

Jenkins rebuild

File: data/json/monsters/monsters.json [hunk 1]
```diff
@@ -4970,7 +4970,7 @@
     "type": "MONSTER",
     "name": "zombeaver",
     "description": "This wall-eyed beast has a large wound on its side, and ribs with some flesh on them are seen through it.  The most prominent feature of this abomination is large incisors in its ominously open mouth, strange black goo is oozing out of it.  Seems like it isn't interested in trees anymore and is looking for some warm fresh to feed.",
-    "default_faction": "zombie",
+    "default_faction": "zombie_aquatic",
     "bodytype": "bear",
```
Comment (Night-Pryanik) [inline data/json/monsters/monsters.json]:
```suggestion
    "bodytype": "dog",
```

File: data/json/monsters/monsters.json [hunk 2]
```diff
@@ -4970,7 +4970,7 @@
     "type": "MONSTER",
     "name": "zombeaver",
     "description": "This wall-eyed beast has a large wound on its side, and ribs with some flesh on them are seen through it.  The most prominent feature of this abomination is large incisors in its ominously open mouth, strange black goo is oozing out of it.  Seems like it isn't interested in trees anymore and is looking for some warm fresh to feed.",
```
Comment (Night-Pryanik) [inline data/json/monsters/monsters.json]:
```suggestion
    "description": "This wall-eyed beast has a large wound on its side, and ribs with some flesh on them are seen through it.  The most prominent feature of this abomination is large incisors in its ominously open mouth, strange black goo is oozing out of it.  Seems like it isn't interested in trees anymore and is looking for some warm flesh to feed.",
```

**Comment by kevingranade** [comment]
Eat Flesh

**Pull request merged**